### ISSUE TITLE
cannot see bounding box on output images

### ISSUE THREAD

**Issue by mimslim** [eriklindernoren/PyTorch-YOLOv3#647]
I tested (using detect.py three images from my training set to see what the output would be.   I don't see any bounding boxes, though detect.py says that it has picked the labels with high confidence (0.999):

Namespace(batch_size=1, checkpoint_model='/mydrive/yolov3/checkpoints/yolov3_ckpt_49.pth', class_path='data/custom/classes.names', conf_thres=0.4, image_folder='/mydrive/yolov3/test/samples', img_size=416, model_def='config/yolov3-custom.cfg', n_cpu=0, nms_thres=0.4, weights_path='/mydrive/yolov3/checkpoints/yolov3_ckpt_49.pth')

Performing object detection:
	+ Batch 0, Inference Time: 0:00:00.100228
	+ Batch 1, Inference Time: 0:00:00.082774
	+ Batch 2, Inference Time: 0:00:00.086843

Saving images:
(0) Image: '/mydrive/yolov3/test/samples/test_image_00004.jpg'
	+ Label: Box, Conf: 0.99999
(1) Image: '/mydrive/yolov3/test/samples/test_image_00005.jpg'
	+ Label: Box, Conf: 1.00000
(2) Image: '/mydrive/yolov3/test/samples/test_image_00006.jpg'
	+ Label: Box, Conf: 0.99999

I am running this in a Google Colaboratory, so I don't expect to be able to view images until I download them on to my personal computer.   When I look, though, the  output images look like the training images, but with no bounding box to show efficacy of network.

**Comment by Flova** [comment]
This is strange because the boxes are detected. Maybe print out the coordinates. I just tested the master in this scenario and it worked fine. The plotting is done by matplotlib. Maybe google collab has issues with the rendering backend. 
You could also switch the `plt.savefig(output_path, bbox_inches="tight", pad_inches=0.0)` to a `plt.show()` and show it in the collab.

**Comment by mimslim** [comment]
I wrote something to create the bounding boxes in my output images, and added it to my copy of detect.py.   The information was there.   Must be some issue with Google Colaboratory environment.
